client/trino-jdbc/src/main/java/io/trino/jdbc/AbstractConnectionProperty.java

```diff
@@ -170,6 +171,11 @@ public void validate(Properties properties)
         throw new IllegalArgumentException("value must be 'true' or 'false'");
     };
 
+    protected static final Converter<GSSCredential> GSS_CREDENTIAL_CONVERTER = value -> {
+        //TODO implement
```

Comment:
Looks like we don't need this mechanism and should just took GssCredential as Object from driver properties